### DISCUSSION TITLE
Campus expert application related issue

### DISCUSSION THREAD

**Discussion by Brijesh1963** [community/community#109776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My GitHub developer pack was approved but till now ,on the application section not able to apply for campus expert...
Please look at this matter, last date of apply 29th Feb.

![image](https://github.com/community/community/assets/98037970/6dd3f90c-e9ec-4b02-ad7a-83e0d4b6fb26)

**Comment by queenofcorgis** [comment]
Hi @Brijesh1963 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have locked this post, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
